app/analytics.go

```diff
@@ -205,31 +200,29 @@ func (a *App) GetAnalytics(name string, teamId string) (model.AnalyticsRows, *mo
 			close(sessionChan)
 		}()
 
-		var fileChan store.StoreChannel
-		var hashtagChan store.StoreChannel
+		var fileCount, hashtagCount int64
+		var fileError, hashtagError *model.AppError
 		if !skipIntensiveQueries {
-			fileChan = a.Srv.Store.Post().AnalyticsPostCount(teamId, true, false)
```

Comment:
This is not the expected approach, the expected approach is to use the explicit async here, creating the channel and calling it in a gorouting (like you can see in the example PR). Not only for `fileChan`, for all the channels that you converted here into synchronous calls.